content/docs/dvclive/api-reference/live/get_step.md

```diff
@@ -1,7 +1,5 @@
 # Live.get_step()
 
-Returns the current `step` value.
```

Comment:
Just curious: why did you decide to remove these short descriptions on top? It's kind of nice to read what something does right from the start IMO but maybe I'm biased bc/ that's what we do in all `dvc` refs.

Reply:
Also the longer Definition above isn't super specific, it talks about this fn. very generally along with `set_step`. Should be more direct I think (esp. if we're removing these shorter ones).

Reply:
Makes sense, thanks! Updated